2-mandatory/6-journey-planner.js

```diff
@@ -80,9 +82,12 @@ function getTransportModes() {}
      * Otherwise, returns false
 
   Hint: Use the corresponding array method to decide if an element is included in an array.
-*/
-function isAccessibleByTransportMode() {}
-
+*/function isAccessibleByTransportMode(transportModes, mode) {
+  if(transportModes.includes(mode)){
+    return true;
+  }
+  return false;
+}
```

Comment:
the `.includes` method works well here, to make this more concise you can simply do:
`return transportModes.includes(mode)` so you can eliminate the if/else statement